### PR TITLE
edits to bias delay slides

### LINKED ISSUE #107: additional topics for delay bias slides
I think we might want: 
- To touch on exponential growth making truncation worse
- Mention both can happen at once and this is tricky
- Flag that dynamic bias can mostly be thought about as right truncation (i.e. you may have heard about dynamic bias blah blah). 
These points can be addressed in a new issue.

_Originally posted by @seabbs in https://github.com/nfidd/nfidd/pull/104#pullrequestreview-2112778254_

### PR DESCRIPTION
**Pull request by sbfnk**
Closes #107.

### CODE REVIEW

File: sessions/slides/introduction-to-biases-in-epidemiological-delays.qmd [hunk 1]
```diff
@@ -83,12 +83,18 @@ We are trying to estimate an incubation period. Each arrow represents one person
 
 ![](figures/truncation_final.png)
 
-On the day of analysis we have not observed some delays yet, and these tended to be longer.
+On the day of analysis we have not observed some delays yet, and these tended to be **longer**. This is made worse during periods of **exponential growth**. 
+
+## Censoring and right truncation {.smaller}
+
+- When analysing data from an outbreak in real time, we are likely to have double censoring **and** right truncation, making things worse
+- In the practical we will only look at the two separately to keep things worse
```
Comment (kathsherratt) [inline sessions/slides/introduction-to-biases-in-epidemiological-delays.qmd:91]:
```suggestion
- In the practical we will only look at the two separately to keep things simple
```

Comment (kathsherratt) [reply sessions/slides/introduction-to-biases-in-epidemiological-delays.qmd:91]: I think this might make more sense but feel free to close out.

**Review by kathsherratt — CHANGES_REQUESTED** [review]
Looks good, just a possible typo

**Review by seabbs — APPROVED** [review]
With the fix from @kathsherratt looks good.

**Review by kathsherratt — APPROVED** [review]

**Pull request merged**